scripts/release-binary.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p gitAndTools.hub -i bash
+project=$(basename $(pwd))
+
+nix build -f release -o $project
```

Comment:
Is `-f` some undocumented flag? I can't find it in `man nix-build`. https://www.mankier.com/1/nix-build

Reply:
Don't confuse `nix-build` with `nix build`. Try `nix build --help`

Reply:
Ah, ok. It says:
>   -f, --file <FILE>             evaluate FILE rather than the default

But there is no` release` file in `crossref-verifier`. Is it automatically implicitly mapped to `release/default.nix`?

Also it says:
> Note: this program is EXPERIMENTAL and subject to change.

From my point of view both `nix-build` and `nix build` can build a derivation. Why can't we use `nix-build` here?

Reply:
> But there is no release file in crossref-verifier. Is it automatically implicitly mapped to release/default.nix?

Yes.

> From my point of view both nix-build and nix build can build a derivation. Why can't we use nix-build here?

`nix-build` is old. Generally everyone is encouraged to use newer-style `nix build`. Do you have a really good reason to use `nix-build`?

Reply:
1. `nix-build` seems to be used everywhere.
2. `nix-build` is not said to be EXPERIMENTAL and subject to change, so if we use `nix-build`, chances that one day something will break are lower. And since this file is going to appear in many repos (probably), it would be nice to decrease chances of having to touch it.

But I don't know how EXPERIMENTAL it is and I didn't know such details:
> Generally everyone is encouraged to use newer-style `nix build`.

If you think it is not too unstable, ok, let's use it.